lib/api/include/irods/get_library_features.h

```diff
@@ -0,0 +1,28 @@
+#ifndef IRODS_GET_LIBRARY_FEATURES_H
+#define IRODS_GET_LIBRARY_FEATURES_H
+
+/// \file
+
+struct RcComm;
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/// Returns a JSON string containing information about the connected server's library features.
+///
+/// \param[in]  _comm     A pointer to a RcComm.
+/// \param[out] _features A pointer to a pointer that will hold the results.
```

Comment:
Worth describing the structure of the JSON string?

Reply:
Done.